selftests/functional/test_basic.py

```diff
@@ -715,8 +720,10 @@ def test_kill_stopped_sleep(self):
                               % (self.tmpdir, SLEEP_BINARY))
         proc.read_until_output_matches(["\(1/1\)"], timeout=3,
                                        internal_timeout=0.01)
-        # We need pid of the avocado, not the shell executing it
-        pid = int(process.get_children_pids(proc.get_pid())[0])
+        # We need pid of the avocado process, not the shell executing it
+        avocado_shell = psutil.Process(proc.get_pid())
+        avocado_proc = avocado_shell.children()[0]
+        pid = avocado_proc.pid
```

Comment:
About that:

1) The default mac OS `ps` program is the BSD version, which means it lacks the capabilities to do what avocado wants it to
2) The gnu coreutils collection can be installed of course, but that requires users to actually do that over their default provided tool. I suspect that won't happen.
3) I can fix process.get_children_pids, but that implies establishing a dependency to the excellent `psutil` module. I'd very much like that, since the library is pretty complete and works against the flavors of Unix that we're interested in. However, I'm not sure if RHEL supplies that library without resorting to EPEL. We'd be back into the whole "avocado requiring EPEL"  conundrum.
4) I suppose we could "borrow" the psutil implementation of that function, but that's the sort of stuff that is frowned upon in Fedora, for example.

Since I want to keep this PR short, so that we can merge it now (we don't need to start Travis CI on OSX just yet), I decided to go with the simple solution of using psutil for the unittest. We already use psutil for unittests and that sorts of solves a problem temporarily. I can come back later and fix `get_children_pids` on a later Pull Request.